src/Event/ReadEventRestController.php

```diff
@@ -87,4 +100,42 @@ public function history($cdbid)
 
         return $response;
     }
+
+    /**
+     * @param string $cdbid
+     *
+     * @return string
+     */
+    public function getCalendarSummary($cdbid, Request $request)
+    {
+        $data = null;
+        $response = null;
+
+        $style = ($request->query->get('style') !== null) ? $request->query->get('style') : 'text';
+        $langCode = ($request->query->get('langCode') !== null) ? $request->query->get('langCode') : 'nl_BE';
+        $hidePastDates = ($request->query->get('hidePast') !== null) ? $request->query->get('hidePast') : false;
+        $timeZone = ($request->query->get('timeZone') !== null) ? $request->query->get('timeZone') : 'Europe/Brussels';
+
+        if ($request->query->get('format') !== null) {
+            $format = $request->query->get('format');
+
+            $data = $this->service->getEvent($cdbid);
+            $event = $this->serializer->deserialize($data, Event::class);
+
+            if ($style === 'html') {
```

Comment:
If you want to be entirely future-proof, some validation on style should be added. If you specify now 'tekst', or anything else than 'html', as parameter value for style, you would still get the 'text' summary. Ideally, specifying an unknown style should result in a error response.